### DISCUSSION TITLE
Gpu mining gets nerfed when running chia

### DISCUSSION THREAD

**Discussion by MrRolypoly** [Chia-Network/chia-blockchain#9157] [General]
Hey guys, I’ve been farming chia and mining with NiceHash at the same time, but recently I had to stop farming chia on 2 of my 6 pcs because I would get half the mh for NiceHash on my gpu. I’m using Ryzen 7 and rtx 3090 and the other has ryzen 7 and rtx 3070. The other 4 computers are the exact same (Ryzen 7 and rtx 3070). Any fixes to this? I have been farming chia for a while and have had no issues before. I’m using windows 10 and the latest 1.2.11.0. In task manager it chia is using 0% gpu, but right next to it it says it’s using GPU 0 - 3D for gpu engine. Also my gpu is not running at 100% when I run chia. The fans get really quiet and my card gets really cool. Thank you!

**Comment by Faysk** [comment]
I had this same problem and realized that it consumed half of the main GPU.
But it stopped when I had the user logged in, I downgraded the Nvidia drive to version 461.92 and there was no longer this problem.

https://www.nvidia.com/en-us/drivers/results/171597/

**Reply by MrRolypoly** [reply]
What do you mean by the user logged in?

**Reply by Faysk** [reply]
"login to windows by remote desktop"

**Reply by MrRolypoly** [reply]
Hey I downgraded to the driver you said but now nicehash wont start because it says my graphics driver is too old.

**Reply by Faysk** [reply]
You can try a version above ( https://www.nvidia.com/en-us/drivers/results/177775/ )

Nicehash should have no problem with 461.92.

![image](https://user-images.githubusercontent.com/5545917/141851367-f9e6026f-cd32-42e7-8370-4d4c50c38c1b.png)

https://github.com/nicehash/NiceHashQuickMiner/releases/tag/v0.5.1.3

**Reply by mdrodge** [reply]
One fix would be not to build computers like that. (have separate Rigs and Servers)
All my miners and my famers run linux.
My miners run HiveOS and my farmer runs Unraid with Machinaris
1 farmer with room for 100's of HDD's and each Mining rig has up to 6 GPU's.
In total you'll have less rigs as you'll be using more of the PCIe slots. You can build a mining rig for under $100 out of an old Dell PC or something, as long as it has PCIe slots it doesn't matter about CPU or RAM specs.
You also benefit because no more windows updates and if you need to power cycle your ETH rig it doesn't effect your Chia.

**Comment by SirGlitchAlot** [comment]
This issue is still occurring and really needs to be addressed. People could potentially mistake a coding bug for gpu mining. Rolling back the driver or limiting chia's use case to setups without GPUs or another setup is not acceptable IMO. I want chia to succeed and adoption to increase!

Related:
https://www.reddit.com/r/chia/comments/n6h32o/question_drop_in_mining_performance_if_chia/
https://github.com/Chia-Network/chia-blockchain/issues/4505
https://github.com/Chia-Network/chia-blockchain/issues/9154
